### DISCUSSION TITLE
Close task list with "q"

### DISCUSSION THREAD

**Discussion by yngwi** [stevearc/overseer.nvim#40] [Q&A] [answered]
I would like to be able to close the task list with the "q" key. Is it possible to set this somehow? Thank you in advance for any help.

**Comment by stevearc** [comment, marked as answer]
Yes, the `task_list.bindings` section of the [setup options](https://github.com/stevearc/overseer.nvim/blob/master/doc/reference.md#setup-options) allows you to specify anything that can be passed to `vim.keymap.set`. The following should work for you:
```lua
require("overseer").setup({
  task_list = {
    bindings = {
      q = "<CMD>close<CR>",
    },
  },
})
```

**Reply by yngwi** [reply]
thank you!
